apps/src/aichat/views/modelCustomization/utils.ts

```diff
@@ -21,3 +23,14 @@ export const validateModelId = (modelId: ValueOf<typeof AiChatModelIds>) => {
 
   return availableModelIds[0];
 };
+
+export const extractFieldsToCheckForToxicity = (
```

Comment:
I see that this has to do with model customizations fields, but wonder if would make more sense to put in `aichat/utils` since used by redux and `aichatApi` and not in react files?

Reply:
Yeah makes sense, I debated between the two `utils` files but wasn't sure what exactly the split was 🙂

Reply:
Moved to `aichat/utils`